src/app/pages/private/ListLivesPage/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Table } from 'antd';
+import { useGetLivesQuery } from '../../../api';
+
+const columns = [
+    {
+        title: 'ID',
```

Comment:
text strings should be part of translation text. 
You can check out https://react.i18next.com/